core/process.py

```diff
@@ -1653,8 +1779,14 @@ def __init__(self, job_list, exec_opts, signal_safe, tracer):
         self.tracer = tracer
         self.last_status = 127  # wait -n error code
 
-    def WaitForOne(self):
-        # type: () -> int
+    #def CheckForNotifications(self):
+    #    """
+    #    Called by main_loop::Interactive()
+    #    """
+    #    pid, status = pyos.WaitPid(WNOHANG)
```

Comment:
@andychu do you have plans for this dead code or can i remove?

Reply:
you can remove this